jcl/src/openj9.dtfj/share/classes/com/ibm/java/diagnostics/utils/plugins/impl/ClassScanner.java

```diff
@@ -36,26 +36,39 @@
 import com.ibm.java.diagnostics.utils.plugins.ClassInfo;
 import com.ibm.java.diagnostics.utils.plugins.ClassListener;
 
-public class ClassScanner extends  ClassVisitor {
+public class ClassScanner extends ClassVisitor {
+
 	private ClassInfo info;
 	private Annotation currentAnnotation = null;
-	private final URL url;			//where the class is being scanned from
+	private final URL url; // where the class is being scanned from
 	private final Set<ClassListener> listeners;
-	
+
 	public ClassScanner(URL url, Set<ClassListener> listeners) {
-		super(Opcodes.ASM4, null);
+		/*[IF Java15]*/
```

Comment:
Seems we need a persistent issue to keep this updated. Java 16 builds will start happening soon I hope.

Reply:
Java 16 doesn't introduce any new ASM versions as compared with Java 15 right now, but, yes, that might change: I created #9990.